alpakaConfig.cmake

```diff
@@ -462,9 +462,9 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
                     IF(CUDA_VERSION GREATER_EQUAL 10.0)
                         MESSAGE(FATAL_ERROR "Clang versions lower than 8 do not support CUDA 10.0 or greater!")
                     ENDIF()
-                ELSEIF(CMAKE_CXX_COMPILER_VERSION LESS 9.0)
-                    IF(CUDA_VERSION GREATER_EQUAL 10.1)
-                        MESSAGE(FATAL_ERROR "Clang versions lower than 9 do not support CUDA 10.1 or greater!")
+                ELSEIF(CMAKE_CXX_COMPILER_VERSION LESS 10.0)
```

Comment:
This is still a bit off. You need two blocks here and LLVM trunk (10) does not yet add 10.2 support:
```cmake
ELSEIF(CMAKE_CXX_COMPILER_VERSION LESS 9.0)
    IF(CUDA_VERSION GREATER_EQUAL 10.1)
        MESSAGE(FATAL_ERROR "Clang versions lower than 9 do not support CUDA 10.1 or greater!")
    ENDIF()
ELSEIF(CMAKE_CXX_COMPILER_VERSION LESS 10.0)
    IF(CUDA_VERSION GREATER_EQUAL 10.2)
        MESSAGE(FATAL_ERROR "Clang versions lower than 10 do not support CUDA 10.2 or greater!")
    ELSE()
        MESSAGE(WARNING "At the time of release it was UNKNOWN if Clang 10 supports CUDA 10.2!")
    ENDIF()
ENDIF()
```
https://gist.github.com/ax3l/9489132#clang--x-cuda

Reply:
I know, was only a temporary test ;-)

Reply:
yeah, but it is unknown if Clang 10 will support CUDA 10.2, that's why I suggested a warning there ;)